serde_core/Cargo.toml

```
@@ -0,0 +1,56 @@
+[package]
+name = "serde_core"
+version = "1.0.188" # remember to update html_root_url and serde_derive dependency
+authors = ["Erick Tryzelaar <erick.tryzelaar@gmail.com>", "David Tolnay <dtolnay@gmail.com>"]
+build = "build.rs"
+categories = ["encoding", "no-std", "no-std::no-alloc"]
+description = "A generic serialization/deserialization framework"
+documentation = "https://docs.rs/serde"
+edition = "2018"
+homepage = "https://serde.rs"
+keywords = ["serde", "serialization", "no_std"]
+license = "MIT OR Apache-2.0"
+readme = "crates-io.md"
+repository = "https://github.com/serde-rs/serde"
+rust-version = "1.31"
+
+[dev-dependencies]
+serde_derive = { version = "1", path = "../serde_derive" }
+serde = {version = "1", path = "../serde"}
+
+[lib]
+doc-scrape-examples = false
+
+[package.metadata.playground]
+features = ["derive", "rc"]
```

There is no `derive` feature on this crate.
```suggestion
features = ["rc"]
```